src/Plugin/PasswordConstraint/MinimumLength.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Drupal\password_enhancements\Plugin\PasswordConstraint;
+
+use Drupal\password_enhancements\Plugin\PasswordConstraintPluginInterface;
+
+/**
+ * Upper-case password constraint plugin.
+ *
+ * @PasswordConstraint(
+ *   id = "minimum_length",
+ *   name = @Translation("Minimum length"),
+ *   description = @Translation("Checks if the password has at least a specified number of character of any type."),
+ *   unique = TRUE,
+ *   jsLibrary = "password_enhancements/plugin.minimum_length",
+ * )
+ */
+final class MinimumLength extends MinimumCharacters implements PasswordConstraintPluginInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultDescriptionSingular(): string {
+    return 'Add at least one letter.';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultDescriptionPlural(): string {
+    return 'Add @minimum_characters more letters.';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validate(string $value): bool {
+    return parent::validateValue($value);
```

Comment:
Do not call `parent` directly. Call `$this->validateValue()` and it will call the parent if needed.

Reply:
Plus move this method to the parent MinimumCharacters class so you do not need to repeat yourself in every child classes - except SpecialCharacter that requires custom handling.